core/arch/arm/plat-imx/platform_config.h

```diff
@@ -52,67 +53,10 @@
 #error "LPAE not supported for now"
 #endif
 
-#define CFG_TEE_CORE_NB_CORE		1
-
-#define CONSOLE_UART_BASE	(UART1_BASE)
-
 /* For i.MX6 Quad SABRE Lite and Smart Device board */
 
 #elif defined(CFG_MX6Q) || defined(CFG_MX6D) || defined(CFG_MX6DL) || \
 	defined(CFG_MX6S)
-
-
-/* Board specific console UART */
-#if defined(PLATFORM_FLAVOR_mx6qsabrelite)
-#define CONSOLE_UART_BASE		UART2_BASE
-#endif
-#if defined(PLATFORM_FLAVOR_mx6qsabresd)
-#define CONSOLE_UART_BASE		UART1_BASE
-#endif
-#if defined(PLATFORM_FLAVOR_mx6dlsabresd)
-#define CONSOLE_UART_BASE		UART1_BASE
-#endif
-#if defined(PLATFORM_FLAVOR_mx6qhmbedge) || \
-	defined(PLATFORM_FLAVOR_mx6dhmbedge) || \
-	defined(PLATFORM_FLAVOR_mx6dlhmbedge) || \
-	defined(PLATFORM_FLAVOR_mx6shmbedge)
-#define CONSOLE_UART_BASE		UART1_BASE
-#endif
-
-
-/* Board specific RAM size */
-#if defined(PLATFORM_FLAVOR_mx6qhmbedge)
-#define DRAM0_SIZE                      0x80000000
-#endif
-#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
-	defined(PLATFORM_FLAVOR_mx6qsabresd) || \
-	defined(PLATFORM_FLAVOR_mx6dlsabresd) || \
-	defined(PLATFORM_FLAVOR_mx6dhmbedge) || \
-	defined(PLATFORM_FLAVOR_mx6dlhmbedge)
-#define DRAM0_SIZE			0x40000000
-#endif
-#if defined(PLATFORM_FLAVOR_mx6shmbedge)
-#define DRAM0_SIZE                      0x20000000
-#endif
-
-/* Core number depends of SoC version. */
-#if defined(CFG_MX6Q)
-#define CFG_TEE_CORE_NB_CORE		4
-#endif
-#if defined(CFG_MX6D) || defined(CFG_MX6DL)
-#define CFG_TEE_CORE_NB_CORE		2
-#endif
-#if defined(CFG_MX6S)
-#define CFG_TEE_CORE_NB_CORE		1
-#endif
-
-/* Common RAM and cache controller configuration */
-#define DDR_PHYS_START			DRAM0_BASE
-#define DDR_SIZE			DRAM0_SIZE
-
-#define CFG_DDR_START			DDR_PHYS_START
-#define CFG_DDR_SIZE			DDR_SIZE
-
 /*
```

Comment:
would you be able to move this below into a config file in the config folder to be consistent with the rest of the configuration.
this could be a config_imx6qdl.h for instance.

Another comment from Jerome was that we could also get rid of the config_ prefix in all those files.

Reply:
The L2 related things could be in a new pull request. This patch is mostly for the DDR related configuraiton. I could not see comments from Jerome.

Reply:
Indeed, I remember commenting that `config/config_something.h` is redundant and would be better written `config/something.h` if possible.

Reply:
@jforissier  I do not want to put all things into one patch, but I could rename or remove the files in a new patch in this patchset.

Reply:
@MrVan sure